packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -232,7 +232,7 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
     },
     {
       authToken,
```

Comment:
please check this ide error 
![Screenshot 2024-11-18 at 3 42 51 PM](https://github.com/user-attachments/assets/75ddae12-dcbd-4991-955b-aed14393a3d2)